miio/airconditioningcompanionMCN.py

```diff
@@ -40,11 +40,10 @@ class AirConditioningCompanionStatus:
     """Container for status reports of the Xiaomi AC Companion."""
 
     def __init__(self, data):
-        """
-        Device model: lumi.acpartner.mcn02
+        """Device model: lumi.acpartner.mcn02.
 
-        Response of "get_prop, params:['power', 'mode', 'tar_temp', 'fan_level', 'ver_swing', 'load_power']":
-        ['on', 'dry', 16, 'small_fan', 'off', 84.0]
+        Response of "get_prop, params:['power', 'mode', 'tar_temp', 'fan_level',
+        'ver_swing', 'load_power']": ['on', 'dry', 16, 'small_fan', 'off', 84.0]
```

Comment:
Here's similar case where the original was much prettier, and there's probably some more. Time to do some manual adjustments :-)